### DISCUSSION TITLE
AlDente Says No Connection Available to Verify License

### DISCUSSION THREAD

**Discussion by Samuels1950** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#438] [General]
Here's the error message I'm receiving. I do have an internet connection and Pro license. 

<img width="263" alt="Screenshot 2021-11-05 at 20 02 08" src="https://user-images.githubusercontent.com/9150966/140565784-6a390f5b-5ccd-4c2b-9b3f-b075ffbb845b.png">

**Comment by MatthiasKerbl** [comment]
Hi @Samuels1950,

I am sorry you are having trouble with the license check.

As basic as it sounds, did you try to restart AlDente or your MacBook?

Cheers,
Matthias

**Reply by Samuels1950** [reply]
Yep. Done both. Thanks for the quick response.

**Reply by Samuels1950** [reply]
Happened after updating to v 1.09

**Reply by MatthiasKerbl** [reply]
Another possibility is to try it on different wifi, maybe a hotspot. Otherwise, send us a support email with your license and we can check it in the backend.
